### PR TITLE
Enable windows client and loader concourse job

### PR DESCRIPTION
**Pull request by pf-qiu**
1. Enable windows ccl pipeline job, fix configure options.
2. Remove C++ check when orca is disabled, add needed header check.
3. Remove non-exist client docs from msi bundle.

Signed-off-by: Yuan Zhao <yuzhao@pivotal.io>

### CODE REVIEW

File: gpAux/Makefile [hunk 1]
```diff
@@ -161,7 +161,7 @@ APU_CONFIG=--with-apu-config=$(BLD_THIRDPARTY_BIN_DIR)/apu-1-config
 
 CODEGEN_CONFIG=--enable-codegen --with-codegen-prefix=/opt/llvm-3.7.1
 
-win32_CONFIGFLAGS=--with-gssapi --without-libcurl $(APR_CONFIG)
+win32_CONFIGFLAGS=--with-gssapi --without-libcurl --disable-orca $(APR_CONFIG)
```
Comment (jpatel-pivotal) [inline gpAux/Makefile]: Why are we disabling orca here?

Comment (pivotal-mike) [inline gpAux/Makefile]: We don't currently have a Win32 g++ cross compiler that support C++11.  It's easier, as we're just building the clients and loaders, to disable ORCA as a prerequisite (not building the actual server).  This way we remove the C++ requirement.

File: configure.in [hunk 2]
```diff
@@ -1296,13 +1296,14 @@ fi
 ## Header files
 ##
 
+_AC_INCLUDES_DEFAULT_REQUIREMENTS
+
 # Check for Greenplum Query Optimizer (orca) and supporting Greenplum OS header files.
-AS_IF([test "$enable_orca" = yes],
-[ # then
+if test "$enable_orca" = yes; then
```
Comment (pf-qiu) [inline configure.in]: At first look I also think that's the case, but the c++ check will not be enclosed with AS_IF.
This way the whole check for ORCA, including c++ compiler, is controlled by enable_orca.

File: gpAux/Makefile [hunk 3]
```diff
@@ -647,6 +647,11 @@ else
 ifneq "$(FILESET_LIB_GCC)" ""
 	(cd $(CLIENTINSTLOC_LIB)/ && $(TAR) cf - -C $(FILESET_LIB_GCC_LOC) $(FILESET_LIB_GCC) | $(TAR) xpf -)$(check_pipe_for_errors)
 endif
+
+	# ---- copy scripts fileset ----
+	mkdir -p $(CLIENTINSTLOC)
+	cp -f client/scripts/greenplum_$@_path$(SCRIPT) $(CLIENTINSTLOC)/
```
Comment (pf-qiu) [inline gpAux/Makefile]: $@ will expand to client or loader. This line is copied from 43 build.
So the final client folder will contains greenplum_clients_path.sh, while the loader folder will contains greenplum_loaders_path.sh.

File: configure.in (hunk 2 above)
Comment (danielgustafsson) [inline configure.in]: The `if` statement is the same regardless, but the difference is that `AS_IF` expands the macros required by the code in the true branch _before_ running the test. This is why you get a check for the C++ preprocessor before testing if `enable_orca` is set to yes.

File: configure.in [hunk 4]
```diff
@@ -1296,13 +1296,14 @@ fi
 ## Header files
 ##
 
+_AC_INCLUDES_DEFAULT_REQUIREMENTS
```
Comment (danielgustafsson) [inline configure.in]: This is an autoconf internal macro that we shouldn't use, and to that effect I don't see why we would want it in the first place? If there is a reason for testing basic header files here we should use the public API for that, but I'm still not convinced about the need.

Comment (pf-qiu) [inline configure.in]: In fact I'm looking for the one which checks for stdint.h and defines HAVE_STDINT_H.
In the original configure script some lines following "On IRIX 5.3 ..." will do that work so I copied that from headers.m4. I'm also not quite satisfied with this workaround, is there a better alternative for checking stdint, and maybe related headers?

Comment (pf-qiu) [inline configure.in]: @danielgustafsson I found that AC_INCLUDES_DEFAULT will do the same work and that's a public macro.

**Pull request merged**